act-rules-format/act-rules-format.bs

```diff
@@ -236,9 +236,9 @@ Rules that can be used to determine if an accessibility requirement is *satisfie
   </ul></blockquote>
 </aside>
 
-#### <dfn>Secondary Requirements</dfn> #### {#secondary-requirements}
+#### Secondary Requirements #### {#secondary-requirements}
 
-A secondary accessibility requirement is a requirement that is correlated with the rule, but for which the rule is not designed to test. The outcome of the rule impacts the result of the accessibility requirement, but the rule is not intended to test the conformance of that requirement. This correlation often results in some of the rule's test cases not satisfying the secondary accessibility requirement.
+A <dfn data-lt="secondary requirement">secondary accessibility requirement</dfn> is a requirement that is correlated with the rule, but for which the rule is not designed to test. The outcome of the rule impacts the result of the accessibility requirement, but the rule is not intended to test the conformance of that requirement. This correlation often results in some of the rule's test cases not satisfying the secondary accessibility requirement.
```

Comment:
HI @kengdoj 

Thanks for your review. I addressed your suggestions. For these definitions to work properly, I just added this `data-lt` here so that we can still use `[=secondary requirements=]` but keep "secondary accessibility requirements` as the visible definition term.

Let me know if you'd prefer something else.